### DISCUSSION TITLE
Guards ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#27] [Announcements]
# Guards ⚡️ Angular с примерами кода

Route Guards позволяют ограничить доступ к маршрутам на основе определенного условия, например, только авторизованные пользователи с определенным набором прав могут просматривать страницу

https://angdev.ru/doc/angular-routing-guards/

**Comment by AlexMihaaa** [comment]
Данная статья требует обновления. Angular 15 рекомендует использовать гуарды не через классы, а функции
